tests/functional/harris/harris_2d.py

```
@@ -1,40 +1,34 @@
 final_time = time_step * time_step_nbr
-dt = 10 * time_step
-nt = final_time / dt + 1
-timestamps = dt * np.arange(nt)
 
 
-def config():
-    sim = ph.Simulation(
+def default_timestamps():
+    dt = 10 * time_step
+    nt = final_time / dt + 1
+    timestamps = dt * np.arange(nt)
```

Ruff (0.8.2)

Local variable `timestamps` is assigned to but never used

Remove assignment to unused variable `timestamps`

(F841)